R/pkg/R/functions.R

```diff
@@ -635,20 +652,16 @@ setMethod("dayofyear",
             column(jc)
           })
 
-#' decode
-#'
-#' Computes the first argument into a string from a binary using the provided character set
-#' (one of 'US-ASCII', 'ISO-8859-1', 'UTF-8', 'UTF-16BE', 'UTF-16LE', 'UTF-16').
+#' @details
+#' \code{decode}: Computes the first argument into a string from a binary using the provided
+#' character set.
 #'
-#' @param x Column to compute on.
-#' @param charset Character set to use
+#' @param charset Character set to use (one of "US-ASCII", "ISO-8859-1", "UTF-8", "UTF-16BE",
+#'                "UTF-16LE", "UTF-16").
```

Comment:
Not a big deal as they contain same information. So, just rather a weak opinion - it'd be nicer if we match this to Scala/Python too IMHO or just leave as is. It's also fine to me as is.